src/development/verkle-replay.md

```diff
@@ -0,0 +1,140 @@
+# Verkle replay
+
+## Context
+
+This data is gathered replaying ~200k historical blocks around the Shanghai fork. While they do not give a perfect view of how execution will behave once EIP-4762 is active, it still gives a good indication of what to expect, and how the spec should evolve.
+
+## Witnesses
+
+This section explores multiple dimensions of the execution witness design.
+
+### Size
+
+Research has been conducted, in order to determine the best structure for a witness. We tried four methods:
+
+ * The first one, as specified in the [current version of the consensus spec](https://github.com/ethereum/consensus-specs/blob/dev/specs/_features/eip6800/beacon-chain.md#executionwitness), referred to as **type 0**. This approach uses the SSZ type `Optional[T]` , that is yet to be officially included in the spec.
+ * A first variation of the spec, in which prestate values are grouped together in their own list, and poststate values are grouped in their own list as well. This is referred to as **type 1**.
+
+```python
+class SuffixStateDiffs(Container):
+    suffixes: bytes
+    # None means not currently present
+    current_values: List[bytes]
+    # None means value not updated
+    new_values:     List[bytes]
+```
+Type 1 serialization container
```

Comment:
Do we need these `Type X serialization container` "legend" when the previous sentence already says `This is referred as **type X**`? (applies to all types)

If you want to keep them, I'd suggest putting them in italics to signal is referring to the thing above and not being a loose sentence.

Reply:
it's left from notion, I'll remove it